consensus/currency.go

```diff
@@ -94,6 +95,12 @@ func (c Currency) RoundDown(n uint64) (y Currency) {
 	return
 }
 
+// ContractTax returns the amount of Currency that would be taxed from a
+// contract payout with value c.
+func (c Currency) ContractTax() Currency {
+	return c.MulFloat(SiafundPortion).RoundDown(SiafundCount)
+}
```

Comment:
I'd rather have ContractTax be a function of a FileContract. Also, it should be named FileContractTax, because in the scheme of cryptocurrencies contracts have a lot of meanings.